axelrod/interaction_utils.py

```diff
@@ -135,6 +135,71 @@ def compute_normalised_state_distribution(interactions):
     return normalized_count
 
 
+def compute_state_to_action_distribution(interactions):
+    """
+    Returns the count of each state to action
+    for a set of interactions.
```

Comment:
I think this needs a little more explanation as to what the function does. It wasn't clear to me until I looked at how it's being used in the result set. Perhaps an example would be useful?